### ISSUE TITLE
OK status is printed as ERROR for some commands in version 1.3.3

### ISSUE THREAD

**Issue by fdanapfel** [henols/firestarter_app#10]
After updating the app to version 1.3.3 I've noticed that some commands now report an OK status as "ERROR" and the status is printed twice:
```
[frank@Mac:~]% pipx upgrade firestarter 
upgraded package firestarter from 1.3.0 to 1.3.3 (location: /Users/frank/.local/pipx/venvs/firestarter)
[frank@Mac:~]% firestarter --version     
Firestarter version: 1.3.3
[frank@Mac:~]% firestarter read M27C1001 M27C1001_2.bin
Reading EPROM M27C1001, saving to M27C1001_2.bin
Address: 0x1FE00 - 0x20000 |████████████████████████████████████████████████████████████| 100.0% - 14.82s

Read complete in: 14.82 seconds
Data saved to M27C1001_2.bin
ERROR: OK                                    <====
ERROR: Unknown state: OK.       <====
[frank@Mac:~]% firestarter fw                          
Reading firmware version...
Current firmware version: 1.3.4, for controller: uno
You have the latest firmware version: 1.3.4, for controller: uno
ERROR: OKstate":13}                               <===
ERROR: Unknown state: OKstate":13}.  <===
```
Also not sure what the `state":13}` is supposed to mean in the result of the "fw" command?

Would be nice if this could be fixed so tha there is only one line showing the result of the command and also that "OK" is not reported as an error.

**Comment by fdanapfel** [comment]
Looks like this has been fixed for the `fw` command in version 1.3.4, but for the `read` command it still returns `ERROR: OK` and `ERROR: Unknown state: OK`:
```
[frank@Mac:~]% pipx upgrade firestarter                     
upgraded package firestarter from 1.3.3 to 1.3.4 (location: /Users/frank/.local/pipx/venvs/firestarter)
[frank@Mac:~]% firestarter fw                                       
Reading firmware version...
Current firmware version: 1.3.4, for controller: uno
You have the latest firmware version: 1.3.4, for controller: uno
[frank@Mac:~]% firestarter blank M27C1001                           
ERROR: Memory is not blank, at 0x000000, value: 0x00
Blank check failed for M27C1001
[frank@Mac:~]% firestarter read M27C1001 M27C1001_3.bin             
Reading EPROM M27C1001, saving to M27C1001_3.bin
Address: 0x1FE00 - 0x20000 |████████████████████████████████████████████████████████████| 100.0% - 14.82s

Read complete in: 14.82 seconds
Data saved to M27C1001_3.bin
ERROR: OK
ERROR: Unknown state: OK
```
This happens on both macOS (Intel iMac) and on Raspberry Pi OS (RPI4) btw.

**Comment by fdanapfel** [comment]
Looks like this is fixed with version 1.3.6 and fw 1.3.5:
```
[frank@Mac:~]% pipx upgrade firestarter
upgraded package firestarter from 1.3.2 to 1.3.6 (location: /Users/frank/.local/pipx/venvs/firestarter)
[frank@Mac:~]% firestarter fw --install  
Reading firmware version...
Current firmware version: 1.3.4, for controller: uno
New firmware version available: 1.3.5, for controller: uno
Installing firmware...
Downloading firmware...
Flashing firmware...
Firmware successfully updated.

[frank@Mac:~]% firestarter read M27C1001 
Reading EPROM M27C1001, saving to M27C1001.bin
Address: 0x1FE00 - 0x20000 |████████████████████████████████████████████████████████████| 100.0% - 14.82s

Read complete in: 14.83 seconds
Data saved to M27C1001.bin
[frank@Mac:~]%
```

**Comment by henols** [comment]
Thanks for reporting and feedback!
Henrik
